animation_nodes/nodes/gpencil/gp_stroke_info.py

```diff
@@ -0,0 +1,40 @@
+import bpy
+from ... data_structures import DoubleList, Vector3DList
+from ... base_types import AnimationNode, VectorizedSocket
+
+class GPStrokeInfoNode(bpy.types.Node, AnimationNode):
+    bl_idname = "an_GPStrokeInfoNode"
+    bl_label = "GP Stroke Info"
+
+    def create(self):
+        self.newInput("GPStroke", "Stroke", "stroke")
+
+        self.newOutput("Vector List", "Points", "vertices")
+        self.newOutput("Float List", "Strengths", "strengths")
+        self.newOutput("Float List", "Pressures", "pressures")
+        self.newOutput("Float List", "UV-Rotations", "uvRotations")
+        self.newOutput("Float", "Line Width", "lineWidth")
+        self.newOutput("Boolean", "Cyclic", "drawCyclic")
+        self.newOutput("Boolean", "Start Cap", "startCapMode")
+        self.newOutput("Boolean", "End Cap", "endCapMode")
+        self.newOutput("Integer", "Material Index", "materialIndex")
+
+        visibleOutputs = ("Points", "Strengths", "Pressures", "Line Width", "Material Index")
+        for socket in self.outputs:
+            socket.hide = socket.name not in visibleOutputs
```

Comment:
I would just hide theme in the `newOutput` call.

Reply:
Changed.